.github/workflows/ubiquibot-config.yml

```diff
@@ -0,0 +1,39 @@
+---
+baseMultiplier: 1000
+timeLabels:
+- name: 'Time: <1 Hour'
+  weight: 0.125
+  value: 3600
+  target: 'Price: 12.5+ USD'
+- name: 'Time: <1 Day'
+  weight: 1
+  value: 86400
+  target: 'Price: 100+ USD'
+- name: 'Time: <1 Week'
+  weight: 2
+  value: 604800
+  target: 'Price: 200+ USD'
+- name: 'Time: <2 Weeks'
+  weight: 3
+  value: 1209600
+  target: 'Price: 300+ USD'
+- name: 'Time: <1 Month'
+  weight: 4
+  value: 2592000
+  target: 'Price: 400+ USD'
+priorityLabels:
+- name: 'Priority: 0 (Normal)'
+  weight: 1
+  target: 'Price: 100+ USD'
+- name: 'Priority: 1 (Medium)'
+  weight: 2
+  target: 'Price: 200+ USD'
+- name: 'Priority: 2 (High)'
+  weight: 3
+  target: 'Price: 300+ USD'
+- name: 'Priority: 3 (Urgent)'
+  weight: 4
+  target: 'Price: 400+ USD'
+- name: 'Priority: 4 (Emergency)'
+  weight: 5
+  target: 'Price: 500+ USD'
```

Comment:
These are special time values that will be added to the issue/bounty if you only set a Priority level, but don't have a time label yet. Sometimes this is nice to ask the bounty hunter how long they think it'll take, and you can just focus on setting the priority level of the task.